projects/addon-editor/constants/tiptap-editor-styles.ts

```diff
@@ -51,7 +51,7 @@ export const tiptapEditorStyles = `
     counter-reset: counter;
 }
 
-.tui-editor-socket ol li p:first-child:before {
+.tui-editor-socket ol li:before {
```

Comment:
if the list item contains two paragraphs, the circle should stay before first one

Reply:
fixed